# ID used to say which tray this is
string destination_id
# ID used to say which order and shipment this is
string shipment_type

---
bool success
float32 inspection_result
